docs/projects/fmc-website/components/header-help.mdx

```diff
@@ -0,0 +1,116 @@
+</div>
+```
+3. Now let's open **script.js**.
+
+4. We have used if-else statement here, if darkMode is equal to false, it means the dark mode is currently off. In that case, the darkModeProp() function is called. This function sets the style properties to define the dark mode appearance, updates the mode text to "LIGHT", changes the mode icon to a sun, and sets darkMode to true and uses the parameters given in the code. If darkMode is not equal to false it means dark mode is currently on. 
+
+``` js title= "script.js" {9-23} showLineNumbers
+const root = document.documentElement.style;
+
+document.addEventListener('DOMContentLoaded', function() {
+	let btnMode = document.getElementById('btn-mode');
+	let modetext = document.getElementById("mode-text");
+	const modeicon = document.getElementById("mode-icon")
+	let darkMode = false;
+
+	btnMode.addEventListener('click', function() {
+		if (darkMode == false) {
+			darkModeProp();
+		  } else {
+			lightModeProp();
+		  }
+	} );
+	function darkModeProp() {
+		root.setProperty("--lm-bg", "#131414");
+		// It's followed by similar parameters
+		modetext.innerText = "LIGHT";
+		modeicon.src = "./assets/icons/sun-icon.svg";
+		root.setProperty("--lm-icon-bg", "brightness(1000%)");
+		darkMode = true;
+		localStorage.setItem("dark-mode", true);
+	}
+
+```
+:::tip
+In the very same way we have also created for how to switch the user interface back to light mode, just changed the paramters values.
+::: 
+
+###  Help
+
+1. The **Need Help** or the **Click here to see instructions** is provided just below the fill my cycle.
+
+2. This is a clickable button which when the user clicks takes him directly to the **instructions components**
+
+3. Now lets understand it's code by opening **index.html** file
+
+4. Overall, the code represents a row with a form that has a heading saying "Need Help?" and a button when the user clicks leads him to instruction part.
+
+5. The div class sets the styling and padding of the **Need Help** text , this text is followed by a clickable button.
+
+``` html title= "index.html"{} showLineNumbers
+<div class="row">
+		<div class="col-sm-6 col-sm-offset-3 col-xs-12">
+			<form class="form-inline" action="" onsubmit="return false">
+				<div class="row">
+					<h5 class="mrg">Need Help? <button class="btn btn-success btn-xs instr-ref">Click here to see instructions</button></h5>
+				</div>
+``` 
+
+After adding the *fill my cycle header*, *theme changer button* and *need help button* the header will look something like this
+
+import header from "/docs/projects/fmc-website/assets/cycle.png";
+
+<figure>
+    <img src={header} style={{ border: "2px solid gray"}} />
+    <center><b><figcaption>Header Section</figcaption></b></center>
+</figure>
+
+_The [**Header Section**](header) is succesfully executed! Let's move on and see how we created the [**Definig cycles**](defining-cycle) of the **TNP Website**._
```

Comment:
- Defining Cycle is wrong
- TNP Website nhi hai Fill My cycle Website hai and also link the website in the text!